index.html

```diff
@@ -0,0 +1,89 @@
+    <meta name="author" content="Arthur Emanuel Guerra Iturres" />
+    <link
+      rel="icon"
+      href="src/assets/media/tab-app-icons/illustration-tab-icon.png"
+    />
+    <link
+      rel="stylesheet"
+      href="src/assets/styles/main-stylesheets/index.css"
+    />
+    <title>Arthur.Iturres | Portfolio</title>
+  </head>
+  <body id="go-top">
+    <nav id="my-navbar" class="my-navbar">
+      <ul class="hidden">
+        <li>
+          <a class="my-navbar__link" href="#go-top">Portfolio</a>
+        </li>
+        <li><a class="my-navbar__link" href="#about-section">About</a></li>
+        <li><a class="my-navbar__link" href="#contact-section">Contact</a></li>
+      </ul>
+      <span class="close-my-navbar__btn my-btn hidden" data-open-my-navbar>
+        <i class="fa-solid fa-x"></i>
+      </span>
+      <div class="open-my-navbar">
+        <span class="open-my-navbar__btn my-btn" data-close-my-navbar>
+          <i class="fa-solid fa-bars"></i>
+        </span>
+      </div>
+    </nav>
+    <header class="my-header">
+      <div class="my-header__title--wrapper">
+        <h1 class="my-header__title">Hey there.</h1>
+        <h2 class="my-header__title">I'm Arthur.</h2>
+      </div>
+      <h3 class="my-header__subtitle">I'm a software developer</h3>
+      <p class="my-header__intro">
+        I can help you build a product, feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don't hesitate to contact me.
+      </p>
+      <nav class="my-header__nav">
+        <ul>
+          <li>
+            <a href="#" class="my-header__link"
+              ><img
+                src="src/assets/media/sections-illustrations/social-icons/twitter.png"
+                alt=""
+                class="link__icon" /></a
+            ><a href="#" class="my-header__link"
+              ><img
+                src="src/assets/media/sections-illustrations/social-icons/linkedin.png"
+                alt=""
+                class="link__icon" /></a
+            ><a href="#" class="my-header__link"
+              ><img
+                src="src/assets/media/sections-illustrations/social-icons/medium.png"
+                alt=""
+                class="link__icon" /></a
+            ><a href="#" class="my-header__link"
+              ><img
+                src="src/assets/media/sections-illustrations/social-icons/github.png"
+                alt=""
+                class="link__icon" /></a
+            ><a href="#" class="my-header__link"
+              ><img
+                src="src/assets/media/sections-illustrations/social-icons/angellist.png"
+                alt=""
+                class="link__icon"
+            /></a>
+          </li>
+        </ul>
+      </nav>
+      <a href="#recent-works" class="my-header__btn go-down-btn my-btn">
+        <i class="fa-solid fa-chevron-down"></i>
+      </a>
+    </header>
```

Comment:
- [ Required ] I think it would be better if this part of the webpage is wrapped in a `section` tag instead of the `header` tag, this part is supposed to be a logical standalone section on your portfolio and semantically it would be nice to wrap it in a `section` tag. Please see this link for [more details >>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section)

Reply:
Again yes, will do. After what you have said and after reading the MDN doc, it makes more sense. thank you!